src/modules/docker/DockerTable.tsx

```diff
@@ -67,28 +69,30 @@ export default function DockerTable({
           />
         </td>
         <td>{element.Names[0].replace('/', '')}</td>
-        <td>{element.Image}</td>
-        <td>
-          <Group>
-            {element.Ports.sort((a, b) => a.PrivatePort - b.PrivatePort)
-              // Remove duplicates with filter function
-              .filter(
-                (port, index, self) =>
-                  index === self.findIndex((t) => t.PrivatePort === port.PrivatePort)
-              )
-              .slice(-3)
-              .map((port) => (
-                <Badge key={port.PrivatePort} variant="outline">
-                  {port.PrivatePort}:{port.PublicPort}
+        {width > 576 ? <td>{element.Image}</td> : null}
+        {width > 576 ? (
```

Comment:
- Use breakpoints provided by the Mantine Theme object
- Consider using a more suitable hook like https://mantine.dev/hooks/use-media-query/